packages/api/src/db/ormconfig.ts

```diff
@@ -0,0 +1,28 @@
+import { createMigrationsList, createOrmConfig } from "@comet/cms-api";
+import { EntityCaseNamingStrategy } from "@mikro-orm/core";
+import path from "path";
+
+export const ormConfig = createOrmConfig({
```

Comment:
Here as well

Reply:
I used the api generator to generate the basic files for the 'BrevoContacts' and the api generator needs the ormconfig to be present . Or is there another way to use the api generator? I can remove the orm config after I generated all the files for the future entities.

Reply:
You'll probably need to run the generator in the Demo project and then copy the files to the library. We also do this for migrations: We generate them in Demo and then move it to the library package.

@nsams do you have a better idea for using the generator in a library package?

Reply:
@johnnyomair I removed the files for now, so this does not block this MR. When there is a better solution for this I will adapt it in the future and I will stick to generating in the demo for now. Is this alright with you?

see [fffcb10](https://github.com/vivid-planet/comet-brevo-module/pull/4/commits/fffcb10e3f52e1047598bbae71f6dac9281ab4d5)